bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/EvccHandler.java

```diff
@@ -82,87 +85,90 @@ public void handleCommand(ChannelUID channelUID, Command command) {
             if (groupId == null) {
                 return;
             }
-            String channelGroupId = channelUID.getGroupId();
+            String channelGroupId = groupId;
```

Comment:
Redundant to create a var `groupId` and then copy that value to `channelGroupId`.

Remove this line and just rename the variable `groupId` above (line 85) to `channelGroupId`.

Reply:
ack. before I touched this one it was even a null-warning ;-)